index.html

```diff
@@ -0,0 +1,130 @@
+    <link rel="stylesheet" href="css/main.css">
+    <title>Homepage</title>
+  </head>
+  
+
+  <!-- Nav Bar -->
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top ">
+        <a class="navbar-brand" href="#">Kizomba ATX</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav ">
+            <li class="nav-item active px-3">
+              <a class="nav-link" href="index.html">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item px-3">
+              <a class="nav-link" href="KizGetdown.html">KizGetdown</a>
+            </li>
+            <li class="nav-item px-3">
+              <a class="nav-link" href="Classes.html">Classes</a>
+            </li>
+            <li class="nav-item px-3">
+                <a class="nav-link" href="instagram.html">Gram</a>
+              </li>
+          </ul>
+        </div>
+      </nav>
+
+
+
+<!-- Header-->
+<header class ="header">
+  <div class="overlay">
+    <div class="container">
+      
+      <div class="description text-left">
+  <h1> Kizomba House of Austin, TX</h1>
+  <p>  Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing industries for previewing layouts and visual mockups.</p>
+     </div>
+        </div>
+</div>
+    
+</header>
+
+
+    
+<!-- About Community Section -->
+<div class="community">
+  <div class="container">
+    <div class="row">
+
+      <div class="col-lg-8 col-md-6 col-sm-12">
+        <div class ="text text-center">
+        <h1>About ATX Kiz Community</h1>
+        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Cursus euismod quis viverra nibh cras pulvinar mattis. Dui accumsan sit amet nulla facilisi morbi tempus iaculis urna. Eu feugiat pretium nibh ipsum. Sit amet justo donec enim diam vulputate ut pharetra sit. Tellus elementum sagittis vitae et leo duis ut. Morbi tristique senectus et netus et malesuada fames. Sit amet nulla facilisi morbi tempus iaculis urna id. Sollicitudin nibh sit amet commodo nulla facilisi nullam.</p>
+            </div>
+      </div>
+
+      <div class="col-lg-4 col-md-6 col-sm-12">
+        <div class="buttons">
+        <a href="http://" class="button">Events</a>
+        <a href="http://" class="button">Music</a>
+        <a href="http://" class="button">Classes</a>
+        <a href="http://" class="button">Donations</a>
+      </div>
+</div>
+    </div>
+  </div>
+  </div>
+
+
+
+
+<!-- About Kizomba Section -->
+<div class="kizomba">
+<div class="container">
```

Comment:
When something is clearly a section, use `<section>`.

Reply:
I dont need a description of section?

Reply:
No, you don't :)